composer.json

```diff
@@ -22,8 +22,8 @@
         "phpunit/phpunit": "^7.5",
         "phpstan/phpstan-deprecation-rules": "~0.12.0",
         "composer/installers": "^1.6",
-        "drupal/core-recommended": "^8.8@alpha",
-        "drush/drush": "^9.6"
+        "drupal/core-recommended": "^8.9",
+        "drush/drush": "^10"
```

Comment:
- [ ] Revert this change, it caused Drush checks to fail, not needed until D9 support added.